samples/add_vm_nic_to_dvs.py

```diff
@@ -8,8 +8,7 @@
 if you python3.5 add_vm_nic_to_dvs ,occur "certificate verify failed",you can
 change the way to coonnect ,eg:
 python 3.5:
-    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
-    context.verify_mode = ssl.CERT_NONE
+    context = ssl._create_unverified_context()
```

Comment:
Yes, you are right, the property only available since 2.7.9. Can we do following:

context = None
if hasattr(ssl, '_create_unverified_context'):
context = ssl._create_unverified_context()